packages/ignite-basic-structure/templates/Unholy/Transforms/ConvertFromKelvin.js

```diff
@@ -1,14 +1,8 @@
 // @flow
 
-import I18n from 'react-native-i18n'
-
 export default (kelvin: number) => {
   const celcius = kelvin - 273.15
   const farenheit = (celcius * 1.8000) + 32
 
-  if (I18n.t('tempIndicator') === 'F') {
```

Comment:
This was a clever way to display the temperature in Celsius if the user's language was anything but English, but since I18n is now a generator we can't do this by default and besides, not all English speaking countries even use Fahrenheit...